### ISSUE TITLE
Stylistic rule for line breaks in define statement

### ISSUE THREAD

**Issue by ilyavolodin** [cvisco/eslint-plugin-requirejs#34]
I think the rule to control line breaks for define statements would be nice. I know I always struggled with this when I used requireJS. I adopted the rule that if define array had more then 3 items, then I write each item in array as well as in a callback function on the separate line, as such:

``` js
define([
  'Backbone',
  'underscore',
  'jquery',
  'someother'
], function(
  'Backbone',
  '_',
  '$',
  'someother'
) {
  ...
});
```

But if there are 3 or less items, then they can be written on a single line:

``` js
define(['Backbone', 'underscore', 'jquery'], function(Backbone, _, $) {
  ...
});
```

Number of max-params should probably be configurable.

**Comment by cvisco** [comment]
I have definitely run into issues with this as well. A rule, or rules, around line breaks is a good idea. Would you see this case as a more focused "one-dependency-per-line" rule, then? With the option being the number of dependencies after which to turn the rule on?

**Comment by ilyavolodin** [comment]
I think this rule should take 1 argument (which would be minimum number of statements before one dependence per line is enforced). If number of dependencies is less provided argument, then this rule might either enforce single line syntax, or ignore it all together. I think either way would be fine.

**Comment by cvisco** [comment]
Alright, after some consideration, I think a schema like the following would make sense: 

``` js
// "one-dependency-per-line": [2, "always", { "after": 3 }]
module.exports.schema = [
    {
        "enum": ["always", "never"]
    },
    {
        "type": "object",
        "properties": {
            "after": {
                "type": "number"
            }
        },
        "additionalProperties": false
    }
];
```
